README.md

```diff
@@ -235,9 +209,6 @@ The Python bootstrapping process will appear in the log first, followed by the R
 and so on. PyCall should now be able to successfully call Python functions from 
 within the Heroku environment.
 
-NB It is also possible to specify buildpacks within Docker images on Heroku.
```

Comment:
This removal is since the Heroku container deploy feature now only supports plain `Dockerfile` deploys, and not the previous experimental `heroku.yml` + buildpacks feature.